test/paymaster.test.ts

```diff
@@ -68,6 +68,11 @@ describe('EntryPoint with paymaster', function () {
       ownerAddr = await ethersSigner.getAddress()
     })
 
+    it('paymaster should revert on wrong entryPoint type', async () => {
+      await expect(new LegacyTokenPaymaster__factory(ethersSigner).deploy(factory.address, 'ttt', AddressZero))
+        .to.be.revertedWith('')
```

Comment:
@drortirosh don't you get a full revert message here?

Reply:
zero address reverts with empty string. 
added a test with some contract with different supportsInterface.